### DISCUSSION TITLE
List of issues found

### DISCUSSION THREAD

**Discussion by shin258** [ikemen-engine/Ikemen-GO#1936] [Other] [answered]
Hi Again since early July i test the my entire roster and stages and i found issues here my list (include build 07-08-2024 and after):

Major:

- Akiha-MA not work at all (all ikemenGO version) perfectly work in winmugen plus and mugen 1.0/1.1
![Akiha-MA error in ikemengo](https://github.com/user-attachments/assets/c50742bb-97d6-4f45-8f3e-fb62c60fd6af)

- The majority of first gen mugen era have a sound (popping) exclusively with ikemenGO 0.99/dev. video example with oni chizuru (release in 2001-2002)
**Note**: winmugenplus / 1.0/1.1 / Ikemengo 0.99/dev are in 48000hz in audio engine setting. For the video save as the video because gihub don't recognize it due of hevc i think.
**in winmugen plus**:
https://github.com/user-attachments/assets/c6512a54-933b-4304-9e14-00bf9c5f55b7
**in Ikemen GO build (07-08-2024 and after)**:
https://github.com/user-attachments/assets/7b219557-a01d-4cef-9780-e689d946b958
- The tag switch don't work with the char myuu (neko) release in 2002 she is the only char to have this issue (ikemenGO 0.99/dev):
https://github.com/user-attachments/assets/756e6ae8-2945-4bda-b4bc-16687f6bc3f6

Corrected with the very last build (thank you to hard work to have correct this) (related to keepaspect):
- rarely some chars fall of the stage
- the transparency and animation of some have stage purple flash in microsecond
- the MSAA work (x8 max)

If i found something else i can update my post if you want. cordially

**Comment by potsmugen** [comment, marked as answer]
The first one looks like a typo for "sub". I guess for "sub" Mugen only cares about the first S. It's debatable whether we should do the same.

Anyway what I wanted to say is ideally you should post these in the issues section, with one topic for each issue.

**Reply by shin258** [reply]
Hi potsmugen, thanks to have respond me, ok no problem i do that.
